### PR TITLE
Species Sprites + Randomized Numbers for Football Shirt

### PR DESCRIPTION
**Pull request by beesil**
Adds species specific sprites for the football shirt.
Shirt number will now be one of four random variations, picked on spawn: 23 (original), 13, 69, or 56.

image

:cl:
 * imageadd: Adds species sprites for the football shirt.
 * rscadd: The number on the back of the football shirt is randomized from one of four options: 23, 13, 69, or 56.

### CODE REVIEW

**Comment by Irradiation** [comment]
NORF

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -991,4 +991,19 @@
...
+			shirt_number = "23"
+		if(2)
+			shirt_number = "13"
+		if(3)
+			shirt_number = "69"
+		if(4)
+			shirt_number = "56"
+	icon_state = "redfootball_[shirt_number]"
+	item_state = "redfootball_[shirt_number]"
+	_color = "redfootball_[shirt_number]"
```
Comment (jknpj) [inline code/modules/clothing/under/miscellaneous.dm]: Maybe...

```suggestion
	icon_state = "redfootball_[pick(23,13,69,56)]"
	item_state = icon_state
	_color = icon_state
```

Comment (ShiftyRail) [reply code/modules/clothing/under/miscellaneous.dm]: ^ do that

**Review by ShiftyRail — CHANGES_REQUESTED** [review]
See suggested changes

**Comment by beesil** [comment]
that's a much better way of doing it, I was lazy & copy-pasted from the greaser suit

**Review by ShiftyRail — APPROVED** [review]

**Pull request merged**